core-plugins/src/test/java/co/cask/hydrator/plugin/transform/MockTestConfigurer.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.transform;
+
+import co.cask.cdap.api.data.schema.Schema;
+import co.cask.cdap.api.data.stream.Stream;
+import co.cask.cdap.api.dataset.Dataset;
+import co.cask.cdap.api.dataset.DatasetProperties;
+import co.cask.cdap.api.dataset.module.DatasetModule;
+import co.cask.cdap.api.plugin.PluginProperties;
+import co.cask.cdap.api.plugin.PluginSelector;
+import co.cask.cdap.etl.api.PipelineConfigurer;
+import co.cask.cdap.etl.api.StageConfigurer;
+
+import javax.annotation.Nullable;
+
+/**
+ * Mock test configurer used to test configure pipeline's validation of input schema and setting of output shcema.
+ */
+public class MockTestConfigurer implements PipelineConfigurer {
```

Comment:
if you review https://github.com/caskdata/hydrator-plugins/pull/72, there will be a mock package in hydrator-common that we can put this in so it can be used by any of the other modules